### PR TITLE
Add Content-Type to header and remove X-Powered-By

### PR DESCRIPTION
**Pull request by gela-d**

### CODE REVIEW

**Comment by gela-d** [comment]
@tadatuta @tormozz48 @incrop

File: src/blocks/server.blocks/middleware/__html-cache/middleware__html-cache.node.js [hunk 1]
```diff
@@ -30,6 +30,9 @@ modules.define('middleware__html-cache', ['logger', 'constants', 'config'],
                 var raw = fs.createReadStream(pagePath),
                     acceptEncoding = req.headers['accept-encoding'] || '';
 
+                res.contentType('text/html; charset=UTF-8');
+                res.removeHeader('X-Powered-By');
```
Comment (tadatuta) [inline src/blocks/server.blocks/middleware/__html-cache/middleware__html-cache.node.js]: there's `app.disable('x-powered-by')` for it and it should be switched off for everything and not just in cache middleware

**Comment by tadatuta** [comment]
the rest looks good if it works

File: src/blocks/server.blocks/middleware/__html-cache/middleware__html-cache.node.js (hunk 1 above)
Comment (gela-d) [inline src/blocks/server.blocks/middleware/__html-cache/middleware__html-cache.node.js]: @tadatuta why not my variant? It works.

Comment (gela-d) [inline src/blocks/server.blocks/middleware/__html-cache/middleware__html-cache.node.js]: @tadatuta I switch off it on page - https://github.com/bem/bem-site-engine/pull/138/files#diff-9df87d4252d418b443fe2b8310557ddbR103

Comment (gela-d) [inline src/blocks/server.blocks/middleware/__html-cache/middleware__html-cache.node.js]: @tadatuta update PR with `app.disable('x-powered-by')`
@tormozz48 need review

**Comment by tavriaforever** [comment]
Check block examples plz before merge

**Comment by gela-d** [comment]
@tavriaforever It's ok.

**Pull request merged**